tests/python/test_interface.py

```diff
@@ -29,33 +31,64 @@ def test(
     format,
     input_data_files,
     input_shape,
+    output_name,
     ref_data_files,
+    it,
+    debug,
+    log_level,
 ):
     import numpy as np
 
+    if debug:
+        log_level = logging.DEBUG
     inputs = []
     ref_outs = []
     for in_file in input_data_files:
-        inputs.append(np.loadtxt(in_file, dtype=np.float32))
+        if str(in_file).find("_int64") >= 0:
+            inputs.append(np.loadtxt(in_file, dtype=np.int64))
+        else:
+            inputs.append(np.loadtxt(in_file, dtype=np.float32))
     for in_file in ref_data_files:
         ref_outs.append(np.loadtxt(in_file, dtype=np.float32))
 
-    service = Inference(model_file, input_shape, device, batch_size, format)
+    service = Inference(
+        model_file,
+        input_shape,
+        output_name,
+        device,
+        batch_size,
+        format,
+        debug,
+        log_level,
+    )
     service.Initialize()
-
-    for i in range(1, 2):
-        print("iteration ", i)
-
+    # Warm up, not count to it.
+    results = service.Run(inputs)
+    inference_time = 0
+    ok = True
+    for i in range(1, it + 1):
+        s = time()
         results = service.Run(inputs)
+        t = time()
+        inference_time += t - s
+        print(f"Overall iteration {i} inference time {(t-s)*1000:.2f} ms")
         for idx, data in enumerate(results):
             of = "/tmp/out." + str(idx) + ".txt"
             np.savetxt(of, data, delimiter="\n")
-        ok = np.allclose(ref_outs[idx], results[idx], rtol=1e-3, atol=1e-3)
-        if ok:
-            print("Results Verified")
-        else:
-            print("Incorrect Results")
-            return 1
+            ok = ok and np.allclose(ref_outs[idx], data, rtol=1e-1, atol=1e-1)
+            if not ok:
+                print("Mismatch of results " + str(idx))
+                print(ref_outs[idx] - data)
+                break
+    if ok:
+        print(
+            f"Results Verified, average inference time {(inference_time / it) * 1000:.2f} ms for {it} iterations"
+        )
+    else:
+        print(
+            f"Incorrect Results, average inference time {(inference_time / it) * 1000:.2f} ms for {it} iterations"
```

Comment:
If iteration is terminated before `it` times, then the average inference time here is not correct. This may use variable `i` instead.

Reply:
It won't print average time if result is incorrect. It returns -1